### ISSUE TITLE
Feature Request: Allow trailing commas

### ISSUE THREAD

**Issue by juanjsebgarcia** [SBeator/sort-js-object-keys#21]
It would be great if this package could add trailing commas to the sorted result. It's a rule I have on my workspace and having to add them manually can be a pain. (Though less of a pain that sorting manually!)

**Comment by dmendonez-mdsol** [comment]
I agree. Plus the animated image in the doc shows that it has trailing commas yet the actual usage does not.
